### PR TITLE
Allow custom source directories

### PR DESCRIPTION
**Pull request by aprokop**
This is an example to allow different placement of source directories. @sayerhs If you are ok with it, all scripts in `codes` can be changed to this using `NALU_SOURCE_DIR`, etc.

### CODE REVIEW

**Comment by sayerhs** [comment]
Agreed, that was the intention of the `<PROJECT>_SOURCE_DIR` but I didn't propagate the changes properly to the scripts.

**Comment by aprokop** [comment]
Updated. The only two I am not sure about are tioga-utils, and wind-utils.

File: codes/tioga-utils.bash [hunk 1]
```diff
@@ -25,7 +25,8 @@ exawind_cmake ()
         -DYAML_DIR:PATH=${YAML_CPP_ROOT_DIR} \
         -DNALU_DIR:PATH=${NALU_WIND_ROOT_DIR} \
         ${install_dir} \
-        ${extra_args} ../src
+        ${extra_args} \
+        ${TIOGA_SOURCE_DIR:-..}/src
```
Comment (sayerhs) [inline codes/tioga-utils.bash]: Should be `TIOGA_UTILS_SOURCE_DIR`

**Comment by aprokop** [comment]
Done.

**Pull request merged**